src/result.ts

```diff
@@ -27,4 +27,9 @@ export enum ErrorCode {
     CREATEPROFFESIONAL_FACILITYIDS_REQUIRED = 'CREATEPROFFESIONAL_FACILITYIDS_REQUIRED',
     UPDATEPROFFESIONAL_FACILITYIDS_REQUIRED = 'UPDATEPROFFESIONAL_FACILITYIDS_REQUIRED',
     INTERNAL_SERVER_ERROR = 'INTERNAL_SERVER_ERROR',
+    INVALID_ORDERBY_OPTION = 'INVALID_ORDERBY_OPTION',
+    ORDERBY_OPTION_NOT_FOUND = 'ORDERBY_OPTION_NOT_FOUND',
+    ORDERBY_OPTION_MAX_LIMIT = 'ORDERBY_OPTION_MAX_LIMIT',
+    ORDERBY_FIELD_SELECTED_TWICE = 'ORDERBY_FIELD_SELECTED_TWICE',
+    INVALID_LANGUAGE_CODE = 'INVALID_LANGUAGE_CODE'
```

Comment:
I have added some error codes, but I’m not sure how specific I should be. Should I be using a more general `NOT_FOUND` or the more specific one `ORDERBY_OPTION_NOT_FOUND`? Or `MAX_LIMIT` instead of `ORDERBY_OPTION_MAX_LIMIT`?